new.go

```diff
@@ -25,8 +27,36 @@ const (
 	DefaultTransport = "docker://"
 )
 
+func reserveSELinuxLabels(store storage.Store, id string) error {
+
+	if selinux.GetEnabled() {
+		containers, err := store.Containers()
+		if err != nil {
+			return err
+		}
+
+		for _, c := range containers {
+			if id == c.ID {
+				continue
+			} else {
+				b, err := OpenBuilder(store, c.ID)
+				if err != nil {
```

Comment:
If we can't get the container, do we need to adjust the values in containers somehow, or is that likely happening when we get this error?

Reply:
I have seen this error, when I corrupted the database.  Perhaps this should be a warning, and it should be easier to cleanup.